### DISCUSSION TITLE
How to create hierarchical folders?

### DISCUSSION THREAD

**Discussion by LuiZzz-Menezes** [coddingtonbear/obsidian-local-rest-api#121] [Q&A] [answered]
I tried the way in the attachment below but I always get `404 status` error.
This `notePath` is something like "Level1/Level2/Level3"

![image](https://github.com/coddingtonbear/obsidian-local-rest-api/assets/40703441/d7cf67bf-7c27-474a-a662-34a6e3234821)

**Comment by coddingtonbear** [comment, marked as answer]
I'm sure you've realized your mistake by now, but if you haven't, you may want to take a closer look at what `encodeURIComponent` is doing to your supplied `notePath` and comparing that with the API docs.  I think you'll find that your request doesn't look quite like what you're imagining.
